client/src/main/js/factory/Reserva.js

```diff
@@ -1,23 +1,36 @@
 let sequenceReserva = 1;
 (() => {
-    'use-strict';
+    'use strict';
     /**
      * Factory que representa a entidade de reserva.
      */
     angular.module('agendamentoModulo', []).factory('Reserva', ['$http', '$q', function ($http, $q) {
 
+        const COR_DEFAULT = '', COR_TEXTO_DEFAULT = '',
+              DIA_INDICE = 0, MES_INDICE = 1, ANO_INDICE = 2,
+              HORA_INDICE = 0, MINUTO_INDICE = 1;
+
+        /*
+         * Obs: Provavelmente terá que ser adicionado duas funções privadas para criação de
+         * uma reserva, uma para quando nós iremos montar e outra para quando receber o evento
+         * da diretiva do calendário. Da maneira implementada, serve apenas para quando
+         * estamos montando nós mesmos.
+         */
         function Reserva(data) {
```

Comment:
Que tal fazer desse jeito?

`function Reserva({_id, autor, ...., cor = COR_DEFAULT, corTexto = COR_TEXTO_DEFAULT}) {
// algo....
Object.assign(this, {_id, autor, cor, ...});
}`